src/centralserver/admin-ui/src/main.ts

```diff
@@ -54,8 +61,8 @@ axios.defaults.headers.get.Accepts = 'application/json';
 Vue.use(Router);
 
 new Vue({
+  pinia,
```

Comment:
Why this is at the beginning of the list?   Example at: https://pinia.vuejs.org/getting-started.html#installation hints that the Pinia is at the end of this list. I just want to learn...

Reply:
No real reason. I moved it to first when I was desperately trying to troubleshoot some issue. It didn't help real reason was somewhere else. 
Moved it to last now just so it looks like that example.  In practice it should not make any difference.